hail/www/navbar.css

```diff
@@ -52,14 +52,15 @@
   border-color: inherit;
 }
 
-#hail-navbar #hail-navbar-collapse a {
+#hail-navbar a {
   color: #444;
   text-decoration: none;
   outline: none;
   background: inherit; /*defeat bootstrap focus background*/
+  text-transform: uppercase;
```

Comment:
I see the appearance of navbar items to be in the domain of a css style, but feel free to push back